models/contacts.js

```diff
@@ -1,14 +1,103 @@
-// const fs = require('fs/promises')
+const fs = require('fs/promises');
+const { v4: uuidv4 } = require("uuid");
+const path = require("path");
+const contactsPath = path.join(__dirname, "..", "models", "contacts.json");
 
-const listContacts = async () => {}
+const readContacts = async (pathFile) => { 
+  const data = await fs.readFile(pathFile);
+  return JSON.parse(data);
+};
 
-const getContactById = async (contactId) => {}
+const writeContacts = async (newContacts, pathFile) => { 
+  const data = JSON.stringify(newContacts);
+  await fs.writeFile(pathFile, data, 'utf8');
+};
 
-const removeContact = async (contactId) => {}
+const listContacts = async () => {
+  try {
+  return await readContacts(contactsPath);
```

Comment:
```suggestion
    return await readContacts(contactsPath);
```
dbaj o indentację w kodzie :)